bbc_sale/data/action_automated_emails_magento.xml

```diff
@@ -75,7 +75,7 @@
             <field name="model_id" ref="stock.model_stock_picking" />
             <field name="type">ir.actions.server</field>
             <field name="state">email</field>
-            <field name="condition">object.picking_type_id.id == 7</field>
+            <field name="condition">True</field>
```

Comment:
Does this not lead to too many emails being sent out?

Reply:
The logic for sending the emails is coded in bbc_sale/models/stock_picking.py #46 do_transfer(self).
As soon as an outgoing delivery is transferred the Python-code checks if the location destination is a customer and if there is a sales order related to the delivery. If this is true, server action to send the mail for delivery shipped will be called.